docs/en/ingest-management/integrations/add-integration-to-policy.asciidoc

```diff
@@ -4,8 +4,18 @@
 Policies consist of one or more integrations. To add a new integration to a
 policy:
 
-. In {kib}, go to *Management > Integrations*. Search for and select an
-integration.
+. In {kib}, go to *Management > Integrations*.
++
+Notice that the Integrations page shows {agent} integrations along with other
+types, such as {beats}.
+
+. Scroll down and select *Elastic Agent only* so the view only shows
```

Comment:
so the view ~only~ shows

other non-agent and non-beats integrations are still shown